src/sidebar/host-config.js

```diff
@@ -50,13 +50,45 @@ export default function hostPageConfig(window) {
     'usernameUrl',
   ];
 
+  // Coerces specific config values that we expect to be of specific types
+  // but could be provided in another.
+  const translations = {
+    // Ensures requestConfigFromFrame.ancestorLevel is integer or NaN
+    requestConfigFromFrame: value => {
+      if (typeof value === 'string') {
+        return value;
+      } else {
+        return {
+          ...value,
+          ancestorLevel: Math.round(Number(value.ancestorLevel)),
+        };
+      }
+    },
+    // Ensures openSidebar is boolean
+    openSidebar: value => {
+      if (value === 'false') {
+        return false;
+      }
+      const numericalVal = Number(value);
+      if (!isNaN(numericalVal)) {
+        return Boolean(numericalVal);
+      }
+      return true;
+    },
```

Comment:
It might be nice to create a `toBool(value)` function and then assign `openSidebar: toBool`. That way the boolean convertion could be tested separately from the sidebar-iness